crates/cast/src/lib.rs

```diff
@@ -426,25 +392,6 @@ mod tests {
             .contains("No such block id mariusz! Possible values are pending, latest, block hash (hex) and block number (u64)."));
     }
 
-    #[test]
-    fn test_get_network() {
-        let testnet = get_network("testnet").unwrap();
-        let testnet2 = get_network("testnet2").unwrap();
-        let mainnet = get_network("mainnet").unwrap();
-
-        assert_eq!(testnet, Network::Testnet);
-        assert_eq!(testnet2, Network::Testnet2);
-        assert_eq!(mainnet, Network::Mainnet);
-    }
-
-    #[test]
-    fn test_get_network_invalid() {
-        let net = get_network("mariusz").unwrap_err();
-        assert!(net
-            .to_string()
-            .contains("No such network mariusz! Possible values are testnet, testnet2, mainnet."));
-    }
-
     #[test]
     fn test_generate_salt() {
         let salt = extract_or_generate_salt(None);
```

Comment:
we also might use some additional tests of `chain_id_to_network_name` to actually see that some random testnames can be used?